### PR TITLE
Fix nested routes

### PR DESCRIPTION
**Pull request by DctrG**
- Allow for nested routes
- Move 404 route to bottom (wildcard route should be last)
- Show selected country

### CODE REVIEW

File: src/router/index.ts [hunk 1]
```diff
@@ -36,7 +31,16 @@ const routes: Array<RouteConfig> = [
   {
     path: '/oembed-map',
     name: 'OembedMap',
-    component: () => import(/* webpackChunkName: 'oembed-map' */ '../views/OembedMap.vue'),
+    component: OembedMap,
+    children: [
```
Comment (yurikoval) [inline src/router/index.ts:35]: This causes the root for oembed to redirect to 404.
Also, country-specific pages are rendered using the Default layout when it should be OembedLayout.

**Review by yurikoval — CHANGES_REQUESTED** [review]

File: src/router/index.ts (hunk 1 above)
Comment (mirhamasala) [reply src/router/index.ts:35]: 1. Actually, it seems 404 is coming from my commit `if (!country) { this.$router.push({ name: 'NotFound' }); return; }`

2. I fixed the rendering of the OembedLayout by adding the `name` and it's working

```
children: [
      { path: ':country.html', name: 'OembedMap', component: OembedMap },
    ],
```

File: public/index.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" type="image/x-icon href="<%= BASE_URL %>favicon-192x192.png" />
     <title><%= htmlWebpackPlugin.options.title %></title>
+    <base href="/" />
```
[the remark this answers is not in the document]
Comment (mirhamasala) [reply public/index.html:9]: Without this, the children routes show only a blank screen, and this is the error message in the console that shows up.

Screenshot 2020-08-19 at 11 43 41 AM

It's fixed with these instructions: https://github.com/vuejs/vue-router/issues/619

Another option would be to modify the `vue.config.js` line 11 with:
`? '/en/ultimate-guide-covid19-travel-restrictions-reopening/' : '/';`

**Review by yurikoval — APPROVED** [review]

**Comment by mirhamasala** [comment]
@DctrG, thank you for contributing. ☀️ Merging now! 🚀

**Pull request merged**